.github/ISSUE_TEMPLATE/03_general_feature.yml

```
@@ -0,0 +1,26 @@
+name: 💡 General feature (not including API proposals)
+description: Add a new feature that does not change the public API surface
+title: "[Feature]: "
+labels: [feature, triage]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        You have a general feature request? Cool, thanks for that.
```

Not sure if I'm fond of the term "feature request", maybe it's just me, but it has some demanding tone in my ears.

How about this?
```suggestion
        You have an idea for a new feature? Cool, let's discuss it.
```

---

Do you mean, that one can think: "Oh.. let's request something and the maintainers have to deliver"?

---

Exactly.
I had to lookup "request" in a dictionary to double check the meaning of it.

I'm not strictly against, just wanted bring it up before merging.

---

I appreciate the effort of reviewing 👍🏻

---

> I had to lookup "request" in a dictionary to double check the meaning of it.

If you take google translate: for the word "request" there are more synonyms for "come on do that.. now" than "please can you.."

So.. I am fine with this formulation